lib/context.js

```diff
@@ -96,11 +96,20 @@ api.process = ({activeCtx, localCtx, options}) => {
           'Unsupported JSON-LD version: ' + ctx['@version'],
           'jsonld.UnsupportedVersion',
           {code: 'invalid @version value', context: ctx});
+      } else if(activeCtx.processingMode && activeCtx.processingMode.indexOf('json-ld-1.1') !== 0) {
+        throw new JsonLdError(
+          '@version: ' + ctx['@version'] + ' not compatible with ' + activeCtx.processingMode,
+          'jsonld.ProcessingModeConflict',
+          {code: 'processing mode conflict', context: ctx});
       }
+      rval.processingMode = 'json-ld-1.1';
       rval['@version'] = ctx['@version'];
       defined['@version'] = true;
     }
 
+    // If not set explicitly, set processingMode to "json-ld-1.0"
```

Comment:
Very minor nit, but we typically don't capitalize at the start of a `//` comment. Comments that require lengthy explanations and full sentences get the `/* */` treatment. This is a short comment so here we'd use `//` and not capitalize, i.e.:

```js
// if not set explicitly, set processingMode to "json-ld-1.0"
```